src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/custom.py

```diff
@@ -690,12 +690,20 @@ def list_features(client, resource_provider_namespace=None):
     return client.list_all()
 
 
-def create_policy_assignment(policy, name=None, display_name=None, params=None,
-                             resource_group_name=None, scope=None):
+def create_policy_assignment(policy=None, policy_set_definition=None,
+                             name=None, display_name=None, params=None,
+                             resource_group_name=None, scope=None, sku=None,
+                             not_scopes=None):
+    """Creates a policy assignment
+    :param not_scopes: Space separated scopes where the policy assignment does not apply.
+    """
+    if bool(policy) == bool(policy_set_definition):
+        raise CLIError('usage error: --policy NAME_OR_ID | \
+        --policy-set-definition NAME_OR_ID')
```

Comment:
nit: The formatting here is not quite right. When I run it, I see the following:
```
# az policy assignment create
usage error: --policy NAME_OR_ID |         --policy-set-definition NAME_OR_ID
```

You need to add a couple extra quotes to avoid the line-wrap from showing in the output.
```
raise CLIError('usage error: --policy NAME_OR_ID | '\
                        '--policy-set-definition NAME_OR_ID')
```